src/components/Table/components/HeaderCell/CellContent.jsx

```diff
@@ -7,16 +7,23 @@ import HeaderCellMenu from "./HeaderCellMenu";
 const CellContent = ({
   children,
   isSortable,
+  isAddEnabled,
   sortedInfo,
   onSort,
   isHidable = true,
   onColumnHide,
+  onAddColumn,
+  onColumnDelete,
   column,
   ...headerProps
 }) => {
   const isColumnHidable = isHidable && isPresent(onColumnHide);
+  const isColumnDeletable = isPresent(onColumnDelete) && column?.isDeletable;
   const hasMoreMenu =
-    isSortable || isPresent(column?.description) || isColumnHidable;
+    isSortable ||
```

Comment:
We need to display more menu when `isColumnDeletable` is true, right?

Reply:
`isColumnDeletable` will never be enabled as the sole option it will always be enabled together with `isAddEnabled` that is why I did not add it to this logic. This will be ensured by the column logic passed to the component.